### PR TITLE
V2 update - documentation

### PR DESCRIPTION
**Pull request by frogpad7**

### CODE REVIEW

File: docs/components/button-container.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 ﻿---
 title: Button Container
-summary: The Button Container collects a series of related buttons at bottom of page.
+summary: The Button Container collects a series of related buttons.
 tags: button container
 layout: guide
 eleventyNavigation:
   key: Button Container
   parent: Components
```
Comment (isralduke) [inline docs/components/button-container.md:8]: We should leave “at bottom of page”

File: docs/components/button-container.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
   img: /img/illustrations/illus-button-container.svg
 ---
 ## Best Practices
 
 - Insert Button Container after the other content, but before any footers.
 - See [Buttons](/components/buttons) for guidance on styling and writing buttons.
+- Consider [buttons’ hierarchies](/components/buttons) when assigning button styles.
 - If there is only 1 cautionary button, such as a `.btn-warning` or `btn-danger`, then add the class `ms-auto` to it.
 - If there are 2 or more cautionary buttons, such as a `.btn-warning` or `btn-danger`, then add the class `ms-auto` to the first.
-- Consider [buttons’ hierarchies](/components/buttons) when assigning button styles.
```
Comment (isralduke) [inline docs/components/button-container.md:19]: Leave this in

File: docs/accessibility/headings.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ Indents in the code below help illustrate how Heading tags create an information
   <h2></h2>
 ```
 
-To use Headings properly
+To use headings properly
```
Comment (isralduke) [inline docs/accessibility/headings.md:35]: again with capitalizing named things in Pelican

File: docs/components/collapse.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ eleventyNavigation:
 
 - Collapses give users control over how much information they see at one time.
 - A common use of Collapses is for a “Question and Answer” item on a FAQ page. 
-- Use the native HTML tag for <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details" target="_blank">Details disclosure Element</a> as Collapses whenever possible.
+- Use the native HTML tag for <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details" target="_blank">Details disclosure Element</a> as collapses whenever possible.
```
Comment (isralduke) [inline docs/components/collapse.md:18]: named thing in Pelican

File: docs/components/dividers.md [hunk 5]
```diff
@@ -15,12 +15,12 @@ eleventyNavigation:
 
 - Use Dividers to separate content.
 - Use Dividers sparingly and only when separating content.
-- Major Dividers create strong separation between content. This could be between Form or content sections.
-- Minor Dividers create weak separation between content. This could be between Form or content section with minor differences.
+- Major Dividers create strong separation between content. This could be between form or content sections.
+- Minor Dividers create weak separation between content. This could be between form or content section with minor differences.
```
Comment (isralduke) [inline docs/components/dividers.md:19]: named thing in Pelican

File: docs/components/modal.md [hunk 6]
```diff
@@ -13,16 +13,16 @@ eleventyNavigation:
 
 ## Best Practices
 
-- Use Modals sparingly.
+- Use modals sparingly.
```
Comment (isralduke) [inline docs/components/modal.md:16]: named thing in Pelican

File: docs/components/page-title.md [hunk 7]
```diff
@@ -16,7 +16,7 @@ eleventyNavigation:
 - Page Titles tell the user which page they’re on.
 - Pelican recommends a Page Title for every page.
 - Use the button to provide quick access to one common action for the page.
-- If a page needs more than one button, use a <a href="https://getbootstrap.com/docs/4.6/components/dropdowns/#single-button" target="_blank">Bootstrap Dropdown button</a>.
+- If a page title needs more than one action, use a <a href="https://getbootstrap.com/docs/5.1/components/dropdowns/#single-button" target="_blank">Bootstrap Dropdown button</a>.
```
Comment (isralduke) [inline docs/components/page-title.md:19]: Great catch. but let’s do the named thing in Pelican here too

File: docs/components/pagination.md [hunk 8]
```diff
@@ -13,10 +13,10 @@ eleventyNavigation:
 
 ## Best Practices
 
-- Use Pagination if data, such as on [Tables](/components/table), is long and needs to be split up.
-- Make sure the Pagination clearly shows the current page’s number.
-- Pagination is placed at the bottom of the content which it pages.
-- You may need to adjust columns sizes for best fit to your project.
+- Use pagination if data, such as on [Tables](/components/table), is long and needs to be split up.
```
Comment (isralduke) [inline docs/components/pagination.md:16]: named thing in Pelican

File: docs/components/popovers-tooltips.md [hunk 9]
```diff
@@ -15,7 +15,7 @@ eleventyNavigation:
 
 - Popovers and Tooltips may appear at the top, right, bottom, or left of the connected object.
 - If space is limited, both will automatically adjust their orientation regardless of initial placement.
-- Popovers have a header. It’s useful when explaining things in more detail than a Tooltip.
+- Popovers have a header. It’s useful when explaining things in more detail than a tooltip.
```
Comment (isralduke) [inline docs/components/popovers-tooltips.md:18]: named thing in Pelican

File: docs/components/table.md [hunk 10]
```diff
@@ -14,9 +14,9 @@ eleventyNavigation:
 ## Best Practice
 
 - Tables allows the user to compare data.
-- Use accessible Table markup, like what’s provided.
+- Use accessible table markup, like what’s provided.
```
Comment (isralduke) [inline docs/components/table.md:17]: named thing in Pelican

File: docs/components/toasts.md [hunk 11]
```diff
@@ -14,14 +14,14 @@ eleventyNavigation:
 ## Best Practices
 
 - Put Toasts at the top right of the screen.
-- Toasts use the same [Status Colors](/foundation/status-colors) as Alerts.
-- Status Colors only add visual cues. The content of the Toast must make the meaning obvious to assistive technology users.
+- Toasts use the same [Status Colors](/foundation/status-colors) as alerts.
```
Comment (isralduke) [inline docs/components/toasts.md:17]: named thing in Pelican

File: docs/form-controls/labels-guidance.md [hunk 12]
```diff
@@ -14,7 +14,7 @@ eleventyNavigation:
 ## Best Practices
 
 - Labels are **required** for form controls.
-- Don’t replace Labels with placeholders.
+- Don’t replace labels with placeholders.
```
Comment (isralduke) [inline docs/form-controls/labels-guidance.md:17]: named thing in Pelican

File: docs/form-controls/textarea.md [hunk 13]
```diff
@@ -1,21 +1,21 @@
 ﻿---
 title: Text Area
-summary: Textarea allows users to input multiple lines of text into a form.
+summary: Text area allows users to input multiple lines of text into a form.
```
Comment (isralduke) [inline docs/form-controls/textarea.md:3]: This is both a named thing in Pelican and it’s also the HTML name of the thing. Let’s leave is as Textarea

File: docs/form-controls/using-form-controls.md [hunk 14]
```diff
@@ -19,6 +19,6 @@ Louisiana government agencies will get the best data possible from the user if t
     
 ## Theming
 
-Unlike other Design System elements, Pelican discourages agency theming of forms and their controls. If an agency does theme forms and controls, ensure maximum compliance with <a href="https://www.section508.gov/" target="_blank">Section 508</a> and <a href="https://www.w3.org/TR/WCAG21/" target="_blank">WCAG guidelines</a>.
+Unlike other design system elements, Pelican discourages agency theming of forms and their controls. If an agency does theme forms and controls, ensure maximum compliance with <a href="https://www.section508.gov/" target="_blank">Section 508</a> and <a href="https://www.w3.org/TR/WCAG21/" target="_blank">WCAG guidelines</a>.
```
Comment (isralduke) [inline docs/form-controls/using-form-controls.md:22]: this is a good call

File: docs/form-controls/using-form-controls.md [hunk 15]
```diff
@@ -19,6 +19,6 @@ Louisiana government agencies will get the best data possible from the user if t
...
 
-Please [read about Status colors](/foundation/status-colors/), [UI colors](/foundation/ui-colors/), and [Agency Theming](/foundation/agency-theming/) for more information.
+Please [read about Status Colors](/foundation/status-colors/), [UI colors](/foundation/ui-colors/), and [Agency Theming](/foundation/agency-theming/) for more information.
```
Comment (isralduke) [inline docs/form-controls/using-form-controls.md:24]: named thing in Pelican

File: docs/form-templates/address.md [hunk 16]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Address
-summary: The Address block allows the user to input their address on a form.
+summary: The Address form allows the user to input their address.
```
Comment (isralduke) [inline docs/form-templates/address.md:3]: Good call to replace block with form. But let’s capitalize Form

File: docs/form-templates/change-password.md [hunk 17]
```diff
@@ -7,7 +7,7 @@ eleventyNavigation:
   key: Change Password
   parent: Form Templates
   order: 2
-  excerpt: The Change Password block allows the user to manage their password.
+  excerpt: The Change Password form allows the user to manage their password.
```
Comment (isralduke) [inline docs/form-templates/change-password.md:10]: named thing in Pelican

File: docs/form-templates/name.md [hunk 18]
```diff
@@ -1,13 +1,13 @@
 ﻿---
 title: Name
-summary: The Name form allows the user to input their name on a form.
+summary: The Name form allows the user to input their name.
```
Comment (isralduke) [inline docs/form-templates/name.md:3]: capitalize form please. i skipped over this apparently

File: docs/form-templates/sign-in.md [hunk 19]
```diff
@@ -1,13 +1,13 @@
 ﻿---
 title: Sign In
-summary: The Sign In block allows the user to sign into an application.
+summary: The Sign In form allows the user to sign into an application.
```
Comment (isralduke) [inline docs/form-templates/sign-in.md:3]: capitalize form please

File: docs/foundation/brand-colors.md [hunk 20]
```diff
@@ -1,13 +1,13 @@
 ﻿---
 title: Brand Colors
-summary: Pelican uses Brand colors to help define your own brand.
+summary: Pelican uses brand colors to help define your own brand.
```
Comment (isralduke) [inline docs/foundation/brand-colors.md:3]: named thing in Pelican

File: docs/foundation/status-colors.md [hunk 21]
```diff
@@ -1,13 +1,13 @@
 ﻿---
 title: Status Colors
-summary: Pelican uses Status colors to inform users about what is going on with the system.
+summary: Pelican uses status colors to inform users about what is going on with the system.
```
Comment (isralduke) [inline docs/foundation/status-colors.md:3]: named thing in Pelican

File: docs/foundation/status-colors.md [hunk 22]
```diff
@@ -23,8 +23,8 @@ eleventyNavigation:
       <div class="bg-success rounded-top pd-color-block"></div>
       <div class="card-body">
         <p class="mb-0 fw-bold">Success</p>
-        <p class="mb-0"><code>#0d6211</code></p>
-        <p class="mb-0"><code>$success</code></p>
+        <p class="mb-0">Hex Code: <code>#0d6211</code></p>
```
Comment (isralduke) [inline docs/foundation/status-colors.md:26]: nice catch

File: docs/foundation/typography.md [hunk 23]
```diff
@@ -13,7 +13,7 @@ eleventyNavigation:
 
 ## Typeface
 
-Pelican’s typeface is Public Sans, designed by USWDS. It is also a variable font, so the download sizes are smaller. Because <a href="https://fonts.google.com/specimen/Public+Sans" target="_blank">Public Sans is available from Google Fonts</a>, it’s ready-to-use in Figma and easy to deploy to a web product.
+Pelican’s typeface is Public Sans, designed by USWDS. It is also a variable font, so the download sizes are smaller. Because <a href="https://fonts.google.com/specimen/Public+Sans" target="_blank">Public Sans is available from Google Fonts</a>, it’s ready-to-use and easy to deploy to a web product.
```
Comment (isralduke) [inline docs/foundation/typography.md:16]: Leave in “in Figma” because designers may need this info

File: docs/index.njk [hunk 24]
```diff
@@ -209,7 +209,7 @@ title: Home
           </div>
           <div class="col-10 col-md-6 col-lg-5 text-center text-lg-end">
               <a href="/get-started/" class="btn btn-white btn-lg">
-                Get started<span class="fas fa-arrow-right ms-1" aria-hidden="true"></span>
+                Get Started<span class="fas fa-arrow-right ms-1" aria-hidden="true"></span>
```
Comment (isralduke) [inline docs/index.njk:212]: earlier comment about not capitalizing all words in a button, but i will defer to @rltucker

File: docs/introduction/change-management.md [hunk 25]
```diff
@@ -21,7 +21,7 @@ The Pelican Design System will grow as we discover solutions created by each tea
 1. The OTS Product Delivery Design Team members will create visual assets in Figma which match code output.
 1. Create branch in Pelican for the change.
 1. Write code.
-1. Pull Reviews and peer review process will occur.
+1. Pull review and peer review processes will occur.
```
Comment (isralduke) [inline docs/introduction/change-management.md:24]: is it a Pull Request or a Pull Review?

**Review by isralduke — DISMISSED** [review]
made some comments. will defer to @rltucker though

File: docs/form-controls/textarea.md (hunk 13 above)
Comment (frogpad7) [reply docs/form-controls/textarea.md:3]: Then why is the title of this page "Text Area"?

File: docs/index.njk (hunk 24 above)
Comment (frogpad7) [reply docs/index.njk:212]: Other non-document uses of this include 404 page, home page, and the sitemap page.

File: docs/components/dividers.md [hunk 26]
```diff
@@ -15,12 +15,12 @@ eleventyNavigation:
...
 - Use Dividers sparingly and only when separating content.
-- Major Dividers create strong separation between content. This could be between Form or content sections.
-- Minor Dividers create weak separation between content. This could be between Form or content section with minor differences.
+- Major Dividers create strong separation between content. This could be between Forms or content sections.
+- Minor Dividers create weak separation between content. This could be between Forms or content sections with minor differences.
 - If the divider is outside a white or gray box, but atop the background of the page, it goes edge-to-edge.
 - If the divider is inside a white or gray box, it stays within the padding of the white or gray box.
 - Top borders on [Back and Forward Buttons](/components/back-and-forward-buttons/) go edge-to-edge.
-- Borders on Components such as [Large Progress Indicator](/components/progress/) typically go edge-to-edge.
+- Borders on components such as [Large Progress Indicator](/components/progress/) typically go edge-to-edge.
```
Comment (rltucker) [inline docs/components/dividers.md:23]: Components should be in caps.

File: docs/index.njk (hunk 24 above)
Comment (rltucker) [reply docs/index.njk:212]: I cap all words in a button (except for words like a, an, the when they appear no as first word.

File: docs/_layouts/human-sitemap.njk [hunk 27]
```diff
@@ -37,14 +37,14 @@ layout: default.njk
...
+<div class="feature-cta bg-texture-accent">
   <div class="container">
     <div class="row justify-content-center align-items-center">
       <div class="col-10 col-md-6 text-center text-lg-start">
-        <p class="feature-big-text text-white">Start using Pelican.</p>
+        <p class="feature-big-text text-white">Start using Pelican now.</p>
       </div>
       <div class="col-10 col-md-6 text-center text-lg-end">
-        <a href="/get-started" class="btn btn-white btn-lg">Get started</a>
+        <a href="/get-started" class="btn btn-white btn-lg">Get Started<span class="fas fa-arrow-right ms-1" aria-hidden="true"></span></a>
```
[the remark this answers is not in the document]
Comment (rltucker) [reply docs/_layouts/human-sitemap.njk:47]: @frogpad7 @isralduke We've generally capitalized everything except articles/ linking verbs.

**Review by rltucker — APPROVED** [review]
Approved!

**Pull request merged**